src/robotide/__init__.py

```diff
@@ -33,7 +35,8 @@
 
 errorMessageTemplate = Template("""$reason
 You need to install wxPython 2.8.12.1 with unicode support to run RIDE.
-wxPython 2.8.12.1 can be downloaded from http://sourceforge.net/projects/wxpython/files/wxPython/2.8.12.1/""")
+wxPython 2.8.12.1 can be downloaded from http://sourceforge.net/projects/wxpyt\
+hon/files/wxPython/2.8.12.1/""")
```

Comment:
Cutting URLs like this doesn't make sense. We don't need to follow PEP-8 that pedantically. Cutting the line before the URL would be OK, though.